CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to Eclipse BlueChi
+
+Thanks for your interest in this project.
+
+## Project description
+
+[Eclipse BlueChi](https://projects.eclipse.org/projects/automotive.bluechi) is a deterministic multi-node service
+controller with a focus on highly regulated ecosystems. It is written in C and integrates seamless with
+systemd via its D-Bus API relaying D-Bus messages over TCP for multi-node
+support. BlueChi is built around three components:
+
+- bluechi-controller
+- bluechi-agent
+- bluechictl
+
+## Developer resources
+
+Information regarding source code management, builds, coding standards, and
+more can be found in the [README.developer](./README.developer.md).
+
+## Eclipse Contributor Agreement
+
+In order to be able to contribute to Eclipse Foundation projects you must
+electronically sign the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php).
+
+The ECA provides the Eclipse Foundation with a permanent record that you agree
+that each of your contributions will comply with the commitments documented in
+the Developer Certificate of Origin (DCO). Having an ECA on file associated with
+the email address matching the "Author" field of your contribution's Git commits
+fulfills the DCO's requirement that you sign-off on your contributions.
+
+For more information, please see the [Eclipse Committer Handbook](https://www.eclipse.org/projects/handbook/#resources-commit)
+
+## Contact
+
+Contact the project developers via the project's "dev" list [here](https://accounts.eclipse.org/mailing-list/bluechi-dev).
```

Comment:
"project's "dev" list" -> "project's development mailing list"

Reply:
This was from the eclipse generator, but I agree. Changing.